src/ion.hpp

```diff
@@ -74,16 +77,35 @@ public :
     view current() {
         return iX_;
     }
+
     view reversal_potential() {
         return eX_;
     }
+
     view internal_concentration() {
         return Xi_;
     }
+
     view external_concentration() {
         return Xo_;
     }
 
+    void reset() {
+        // The Nernst equation uses the assumption of nonzero concentrations:
+        EXPECTS(default_internal_concentration> value_type(0));
+        EXPECTS(default_external_concentration> value_type(0));
+        memory::fill(iX_, 0);
+        memory::fill(Xi_, default_internal_concentration);
+        memory::fill(Xo_, default_external_concentration);
+        nernst_reversal_potential(6.3+273.15); // TODO: use temperature specfied in model
```

Comment:
The 6.3 °C temperature now appears as an explicit number in a few places. Until there is a settable temperature somewhere, this at least should sit in a constant somewhere (e.g. `fvm_multicell`). `reset()` can then take the temperature as a parameter.

I'm also a bit worried that there is no way to explicitly set the reversal potential independent of any concentrations, making it hard to obtain the same behaviour as e.g. NEURON's hh mechanism. If this is regarded as out-of-scope, I'll make a ticket with a proposed API.

Reply:
I have put it in `constants.hpp` as `squid_temperature` (a ridiculous name, with the understanding that it i will soon be replaced).